app/src/test/java/com/x8bit/bitwarden/MainViewModelTest.kt

```diff
@@ -1013,6 +1083,7 @@ private fun createMockIntent(
     mockIsMyVaultShortcut: Boolean = false,
     mockIsPasswordGeneratorShortcut: Boolean = false,
     mockIsAccountSecurityShortcut: Boolean = false,
+    mockIsAddTotpLoginItemFromAuthenticator: Boolean = false,
```

Comment:
Wow, this is amazing. Someone did a great job cleaning this up 😆

Reply:
❤️ love when stuff like this happens!